appveyor.yml

```diff
@@ -73,11 +72,12 @@
     assembly_informational_version: "2.2.0.0"
 
   build_script:
-    - msbuild "src\Vlc.DotNet.sln" /logger:"C:\Program Files\AppVeyor\BuildAgent\Appveyor.MSBuildLogger.dll" /p:Configuration=Release;Platform=AnyCPU
-    - ps: nuget pack C:\projects\vlc-dotnet-master\nuget\Vlc.DotNet.Core.Interops.nuspec -version $env:APPVEYOR_BUILD_VERSION
-    - ps: nuget pack C:\projects\vlc-dotnet-master\nuget\Vlc.DotNet.Core.nuspec -version $env:APPVEYOR_BUILD_VERSION
-    - ps: nuget pack C:\projects\vlc-dotnet-master\nuget\Vlc.DotNet.Forms.nuspec -version $env:APPVEYOR_BUILD_VERSION
-    - ps: nuget pack C:\projects\vlc-dotnet-master\nuget\Vlc.DotNet.Wpf.nuspec -version $env:APPVEYOR_BUILD_VERSION
+    - msbuild /t:Restore /Verbosity:Minimal "src\Vlc.DotNet.sln"
+    - msbuild /t:Build /p:Configuration=Release "src\Vlc.DotNet.sln"
+    - msbuild /t:Pack /p:Configuration=Release "src\Vlc.DotNet.Core.Interops\Vlc.DotNet.Core.Interops.csproj"
```

Comment:
Where has the version gone?

Reply:
You can provide a VersionSuffix and a VersionPrefix as MSBuild variables. The .NET Core SDK then uses that for assembly and NuGet package version numbers. I've updated the PR to do that.

For the time being, I disabled automatic assembly version information generation because it conflicts with some of the AssemblyInfo files you already have. It's something you may want to consider enabling later.